### ISSUE TITLE
[Excalibur] Show R0 summary statistics

### ISSUE THREAD

**Issue by sychang** [Recidiviz/covid19-dashboard#227]
Blocked by Rt calculation and UI work

PRD section: https://docs.google.com/document/d/1_jbpbsObUWkakyitmuIMaNKhq1YVU_fpo4j06ip8ets/edit#heading=h.2a9nh9h20rm5

Mock: https://www.figma.com/file/6z1QQW84a59Ul3c17PgWdV/%F0%9F%9A%80Recidiviz-CJStatus-(excalibur)?node-id=1837%3A652

**Comment by daschi** [comment]
Is this a duplicate of this issue ? https://github.com/Recidiviz/covid19-dashboard/issues/239

**Comment by daschi** [comment]
Closing as this was covered in #239
